rhel8/profiles/cis.profile

```diff
@@ -601,87 +601,88 @@ selections:
 
     ### 4.1.9 Ensure discretionary access control permission modification
     ###       events are collected (Scored)
-
+    # NEEDS RULE - https://github.com/ComplianceAsCode/content/issues/5509
     
     ### 4.1.10 Ensure unsuccessful unauthorized file access attempts are
     ###        collected (Scored)
-
+    # NEEDS RULE - https://github.com/ComplianceAsCode/content/issues/5510
```

Comment:
Could we use rules from the audit_file_modification group?

Reply:
Likely! Feel free to self-assign that ticket and review/patch as applicable.